sdk/v2/api_client.go

```diff
@@ -13,13 +12,11 @@ import (
 type APIClient interface {
 	Authx() authx.APIClient
 	Core() core.APIClient
-	System() system.APIClient
 }
 
 type apiClient struct {
-	authxClient  authx.APIClient
-	coreClient   core.APIClient
-	systemClient system.APIClient
```

Comment:
The `system` package and the API clients in it were the result of a previous failure to find the right home for stuff. The `system` package goes away on both the SDK/API client side and API server side and roles, logic, etc. previously found therein all moves to a better home. This is a consistent theme throughout this PR.